test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js

```diff
@@ -105,7 +138,7 @@ test.requestHooks(mockOnlyOneIdp)('should auto redirect to 3rd party IdP login p
   });
 
   // assert redirect to IdP login page eventually
-  await t.expect(await Selector('h1').innerText).eql('A external IdP login page for testcafe testing');
+  await t.expect(Selector('h1').innerText).eql('A external IdP login page for testcafe testing');
```

Comment:
why we needed to remove `await` from here ?

Reply:
When I ran the test locally I got this warning
```Warnings (1):
 --
  You passed a DOM snapshot property to the assertion's 't.expect()' method. The
  property value is assigned when the snapshot is resolved and this value is no
  longer updated. To ensure that the assertion verifies an up-to-date value, pass
  the selector property without 'await'.

     136 |    controller: null,
     137 |    formName: 'success-redirect',
     138 |  });
     139 |
     140 |  // assert redirect to IdP login page eventually
   > 141 |  await t.expect(await Selector('h1').innerText).eql('A external IdP
  login page for testcafe testing');
     142 |  const pageUrl = await ClientFunction(() => window.location.href)();
     143 |  await
  t.expect(pageUrl).eql('http://localhost:3000/sso/idps/facebook-idp-id-123?stateToken=inRUXNhsc6Evt7GAb8DPAA');
     144 |
     145 |});
     146 |`

Reply:
was it working earlier ?

Reply:
It's a warning so it would work/pass anyway. Warning went away once I removed the await.